### DISCUSSION TITLE
Error setting up environment using elevated PowerShell script

### DISCUSSION THREAD

**Discussion by uwemneku** [microsoft/react-native-windows#9803] [Q&A] [answered]
I got a new system running on windows 11 and I'm having issues running the shell script to set up react native windows environment.
```
Set-ExecutionPolicy Unrestricted -Scope Process -Force;
iex (New-Object System.Net.WebClient).DownloadString('https://aka.ms/rnw-deps.ps1')
```
Please I need help resolving this error.

Here's a screenshot of the error I'm getting.
![error image](https://i.ibb.co/zZPPj1G/error.png)

npx react-native info
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 3.03 GB / 15.84 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.18 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.3.1 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
  IDEs:
    Android Studio: Version     2021.1.0.0 AI-211.7628.21.2111.8193401
    Visual Studio: Not Found
  Languages:
    Java: 1.8.0_322 - C:\Program Files\Eclipse Adoptium\jdk-8.0.322.6-hotspot\bin\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.66.4 => 0.66.4 
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**Comment by chrisglein** [comment, marked as answer]
Looks like this may be an issue in the dependencies script. Created this issue to track it: https://github.com/microsoft/react-native-windows/issues/9807

**Comment by uwemneku** [comment]
The issue is fixed. Thank you so much 🙂
